helmfile/50-applications.yaml

```diff
@@ -369,9 +383,9 @@ releases:
   version: 0.1.0
   installed: true
   needs:
-  - opensearch-system/opensearch-master
-  - opensearch-system/opensearch-dashboards
+  - opensearch-system/opensearch-securityadmin
   values:
```

Comment:
Should we add opensearch dashboard here as well? Though I know we have some check in the configurer script as well.

Reply:
Right, I got to eager while cleaning up.